### PR TITLE
feat(pilot-story-globe): move globe down and in

### PR DESCRIPTION
**Pull request by andreashelms**
for #1412

image

### CODE REVIEW

File: src/scripts/components/stories/pilot-story/components/globe/globe.tsx [hunk 1]
```diff
@@ -63,6 +65,16 @@ const Globe: FunctionComponent<Props> = ({
...
   }, [containerRef, globe]);
 
+  useEffect(() => {
+    if (containerRef.current) {
+      containerRef.current.style.top = `${initialPosition.top}%`;
+      containerRef.current.style.left = `${initialPosition.left}%`;
+      containerRef.current.style.right = `${initialPosition.right}%`;
+      containerRef.current.style.bottom = `${initialPosition.bottom}%`;
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (Immo-Be) [inline src/scripts/components/stories/pilot-story/components/globe/globe.tsx:75]: You should be able to memoize `initialPosition` and get rid of the comment here

**Review by Immo-Be — APPROVED** [review]

File: src/scripts/components/stories/pilot-story/components/globe/globe.tsx (hunk 1 above)
Comment (andreashelms) [reply src/scripts/components/stories/pilot-story/components/globe/globe.tsx:75]: Can you please explain how?

Comment (Immo-Be) [reply src/scripts/components/stories/pilot-story/components/globe/globe.tsx:75]: As `initialPosition` is passed down as a prop, you could define it in the parent like 

```
  const memoizedRelativePosition = useMemo(
    () => ({top: 0, left: -100, right: 0, bottom: 0}),
    []
  );
```

Comment (andreashelms) [reply src/scripts/components/stories/pilot-story/components/globe/globe.tsx:75]: Ok, will be removed with next PR.

**Pull request merged**